Resources/templates/responsive/invest/partials/reward_box.php

```diff
@@ -68,6 +68,12 @@ class="form-control input-amount"
                                    title="<?= $this->t('regular-amount') ?>"
                                    aria-labelledby="amount-<?= $reward->id ?>"
                                    required>
+                            <input type="number"
+                                   class="form-control input-amount"
+                                   name="donate_amount"
+                                   value="2"
```

Comment:
This value is present in the configuration yaml of the application. This should be changed to search it in the configuration file through the $this->get_config() function.